spec/features/admin_edits_redirection_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.feature "Admin edits redirection" do
+  scenario "edits URL" do
+    new_url = "https://example.com/new"
+    url = "https://foobar.neocities.org"
+    original_url = "https://original.neocities.org"
+    slug = "binturong"
+    redirection = create(:redirection, slug:, url:, original_url:)
```

Comment:
Is this a shortcut for `create(:redirection, slug: slug, etc)`? I've never seen blank keyword arguments when calling a method.

Reply:
So your first comment was correct!

As for your second comment: Maybe you didn't set the local variable name when testing in IRB, or maybe you ran it in an IRB that was older than v3.1.0?